users/tests/snapshots/snap_test_notifications.py

```diff
@@ -42,46 +42,46 @@
 
 Child name: Jason Berg
 
-Event: Enjoy when one wonder fund nor white.
-Occurrence: 2011-01-05 08:35:11+00:00
+Event: Policy sport available.
+Occurrence: 1983-12-14 10:41:44.139987+00:00
 
-Event: Affect money school military statement.
-Occurrence: 1972-02-01 22:06:34+00:00
+Event: Month score father middle brother station physical very.
+Occurrence: 1989-06-15 22:58:14.498478+00:00
```

Comment:
The result now contains milliseconds! It didn't have those earlier.

Reply:
I think this has changed with Django upgrade and since this seems to affect only to Occurrence time-fiel (datetimefield), I think we can jsut approve this.

https://docs.djangoproject.com/en/4.2/ref/models/database-functions/#django.db.models.functions.Now